### PR TITLE
Import AllocationError from pynetbox.core.query, also add ContentError to handle json failures

### PR DESCRIPTION
**Pull request by markkuleinio**
This imports AllocationError from pynetbox.core.query to the package level in pynetbox/__init__.py.

### CODE REVIEW

**Comment by markkuleinio** [comment]
I see the system includes all the commits in the same pull request, no problem.

Anyway, by adding **ContentError** I'm trying to address #165 that is caused by the server outputting something unexpected, meaning that the server is not a NetBox server at all. `req.ok` is still `True`, so `req.json()` will fail. With ContentError:

```
>>> import pynetbox
>>> netbox = pynetbox.api("http://172.24.24.7/", token="That's a server redirecting to some page")
>>> try:
...     netbox.dcim.devices.all()
... except pynetbox.ContentError as e:
...     print(e)
...
The server returned invalid (non-json) data. Maybe not a NetBox server?
>>>
```

File: pynetbox/__init__.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 from pkg_resources import get_distribution, DistributionNotFound
 
-from pynetbox.core.query import RequestError
+from pynetbox.core.query import RequestError, AllocationError
+from pynetbox.core.query import ContentError
```
Comment (zachmoody) [inline pynetbox/__init__.py]: I think this would be ok on L3 or did the linter complain?

Comment (markkuleinio) [reply pynetbox/__init__.py]: Oh, yes, no complaints now that I joined the lines. I just thought the limit was 72 characters! :-D

Comment (zachmoody) [reply pynetbox/__init__.py]: Nice. Thanks!

**Pull request merged**